v8/payloads/guild.ts

```diff
@@ -114,6 +114,7 @@ export interface APIGuild extends APIPartialGuild {
 	 * Voice region id for the guild
 	 *
 	 * See https://discord.com/developers/docs/resources/voice#voice-region-object
+	 * @deprecated No longer used, `rtc_region` on the channel is now used instead.
```

Comment:
Is the field removed tho?

```suggestion
	 * @deprecated This field has been deprecated in favor of `rtc_region` on the channel.
```

Reply:
it still exists but doesn't do anything afaik